deploy/examples/create-external-cluster-resources.py

```diff
@@ -969,6 +970,16 @@ def get_rbd_node_caps_and_entity(self):
 
         return caps, entity
 
+    def get_healthchecker_caps_and_entity(self):
```

Comment:
hey, these values should not be hardcoded, as with the new script the default caps would change

You should update `create_checkerKey` how  `create_cephCSIKeyring_user` runs, 
 it should first `check_user_exist` and then only create it.